<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VacuumCellWidget</class>
 <widget class="QWidget" name="VacuumCellWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>20</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="lblTitle">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>20</height>
      </size>
     </property>
     <property name="text">
      <string>Bending</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>5</width>
       <height>17</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QELabel" name="lblIMG">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>90</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>20</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="variable" stdset="0">
      <string>$(group):getPressure</string>
     </property>
     <property name="variableSubstitutions" stdset="0">
      <string>group=SRC01-VA-IMG1</string>
     </property>
     <property name="precision">
      <number>3</number>
     </property>
     <property name="useDbPrecision">
      <bool>false</bool>
     </property>
     <property name="addUnits">
      <bool>false</bool>
     </property>
     <property name="notation" stdset="0">
      <enum>QE::Scientific</enum>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer_5">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>10</width>
       <height>14</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QELabel" name="lblIP1">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>70</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>70</width>
       <height>20</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="variable" stdset="0">
      <string>SRC$(cell)-VA-IP$(ip):getPressure</string>
     </property>
     <property name="variableSubstitutions" stdset="0">
      <string/>
     </property>
     <property name="precision">
      <number>1</number>
     </property>
     <property name="useDbPrecision">
      <bool>false</bool>
     </property>
     <property name="addUnits">
      <bool>false</bool>
     </property>
     <property name="notation" stdset="0">
      <enum>QE::Scientific</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QELabel" name="lblIP2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>70</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>70</width>
       <height>20</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="variable" stdset="0">
      <string>SRC$(cell)-VA-IP$(ip):getPressure</string>
     </property>
     <property name="precision">
      <number>1</number>
     </property>
     <property name="useDbPrecision">
      <bool>false</bool>
     </property>
     <property name="addUnits">
      <bool>false</bool>
     </property>
     <property name="notation" stdset="0">
      <enum>QE::Scientific</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QELabel" name="lblIP3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>70</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>70</width>
       <height>20</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="variable" stdset="0">
      <string>SRC$(cell)-VA-IP$(ip):getPressure</string>
     </property>
     <property name="precision">
      <number>1</number>
     </property>
     <property name="useDbPrecision">
      <bool>false</bool>
     </property>
     <property name="addUnits">
      <bool>false</bool>
     </property>
     <property name="notation" stdset="0">
      <enum>QE::Scientific</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QELabel" name="lblIP4">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>70</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>70</width>
       <height>20</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="variable" stdset="0">
      <string>SRC$(cell)-VA-IP$(ip):getPressure</string>
     </property>
     <property name="precision">
      <number>1</number>
     </property>
     <property name="useDbPrecision">
      <bool>false</bool>
     </property>
     <property name="addUnits">
      <bool>false</bool>
     </property>
     <property name="notation" stdset="0">
      <enum>QE::Scientific</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QELabel</class>
   <extends>QLabel</extends>
   <header>QELabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
